Python/mergetwopdf/Readme.MD

```diff
@@ -0,0 +1,23 @@
+# Merge two pdf Files
+
+### Third Party Libraries Required :
+1.PyPDF2
+
+### How to install above Library
+```
+import PyPDF2
+```
+
+### How to use it :
+1. Download or clone the repository
+2. Install Required Libraries
+3. Run merge.py by giving pdf file through command line
+4. you will get the merged pdf in the same directory

```

Comment:
Add proper commands to run the Script. Many people are not familiar with Argument Parsers. This would be helpful for them in the longer run.